.github/workflows/ci.yml

```diff
@@ -305,7 +305,7 @@ jobs:
       - name: Build
         run: |
           rustup target add wasm32-unknown-unknown
-          cargo build --target wasm32-unknown-unknown --no-default-features --features=services-s3
+          cargo build --target wasm32-unknown-unknown --no-default-features --features=services-s3,services-azblob
```

Comment:
Let's extend this like we do for `build_all_platforms`.

Reply:
Okay.

Reply:
We could set a new feature flag for wasm32.

Reply:
> We could set a new feature flag for wasm32.

Why?

Reply:
To group those services that are already available under wasm32?

Reply:
> To group those services that are already available under wasm32?

I feel like this feature flag is meaningless. I don't want to expose arch related feature flag to users.

Adding an array in shell is good enough to me.

Reply:
> I don't want to expose arch related feature flag to users.

For this, I mean: do you want us to have `services-enabled-x86_64`, `services-enabled-wasm32`, `services-enabled-aarch64`?

Reply:
Make sense.

Reply:
> For this, I mean: do you want us to have `services-enabled-x86_64`, `services-enabled-wasm32`, `services-enabled-aarch64`?

It's a bit wired.